pkg/webhook/webhook.go

```diff
@@ -40,5 +42,16 @@ func NewWebhookCommand(stopChan <-chan struct{}) *cobra.Command {
 	cmd.Short = "Start a kubefed webhook server"
 	cmd.Long = "Start a kubefed webhook server"
 
+	versionFlag := false
+	cmd.Flags().BoolVar(&versionFlag, "version", false,
+		"Prints version information for kubefed admission webhook and quits")
+	cmd.PreRun = func(c *cobra.Command, args []string) {
```

Comment:
nit: Do we need to validate if someone gives additional command line options in addition to --version?

Reply:
We don't need to validate for additional arguments because we'd have to do that for all commands, including `-h` for help, which supersedes any option regardless if more options are specified. So this would be consistent with that behavior and any other executable with POSIX style flags. For additional options which are invalid, the `pflag` package will handle those.

Reply:
👍 . Thanks for clarifying that.